chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client_unittest.cc

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "base/test/scoped_feature_list.h"
+#include "brave/components/ai_chat/core/common/features.h"
+#include "components/omnibox/browser/autocomplete_controller.h"
+#include "components/omnibox/browser/autocomplete_match.h"
+#include "components/omnibox/browser/autocomplete_result.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace {
+
+class AutocompleteControllerLeoTest : public testing::Test {
```

Comment:
1. tests should not be in anonymous namespace
2. this file makes all upstream tests implemented in `chrome/browser/autocomplete/chrome_autocomplete_provider_client_unittest.cc` inaccessible, because it doesn't include the upstream file.
3. you should not add a test file that already exists in upstream test suite which we run on CI (`browser_tests`, `unit_tests`) in `brave_*` test suite.

If you want to disable all upstream tests from the file, consider using `brave/test/filters/unit_tests.filter` with a clear comment explaining the reasoning behind it.

I'd say this test file should be at `//brave/browser/autocomplete` and included into `brave_unit_tests` via `unit_tests` target implemented in `//brave/browser/autocomplete/BUILD.gn`.

Reply:
Fixed in 8044f1b72e2e62c3bdd8cd34c66e39fec50bb8d2